tftrt/examples/image-classification/image_classification.py

```diff
@@ -72,9 +77,16 @@ def model_fn(features, labels, mode):
             input_map={'input': features},
             return_elements=['logits:0', 'classes:0'],
             name='')
-        loss = tf.losses.sparse_softmax_cross_entropy(labels=labels, logits=logits_out)
-        accuracy = tf.metrics.accuracy(labels=labels, predictions=classes_out, name='acc_op')
+        if mode == tf.estimator.ModeKeys.PREDICT:
+            predictions = {}
+            with tf.name_scope('Predictions'):
+                predictions['logits'] = logits_out
```

Comment:
I think we can also inline these: 

`tf.estimator.EstimatorSpec(mode=mode, predictions={'logits':logits_out, 'classes':classes_out})`

Reply:
It might be better to only fetch classes to reduce IO.

Reply:
oh yes, that's a good idea.